### PR TITLE
Errors occur on UploadControl demos (close #261)

### LINKED ISSUE #261: Errors occur on UploadControl demos
Errors are produced via playground after click on 'Upload' button
```
"Uncaught JavaScript error Uncaught TypeError: Cannot redefine property: %hammerhead% on page"

"@test"["Select file"] = {
    'Upload': function() {
        var fileInput = $('#ContentHolder_uplImage_TextBox0_Input');
        act.upload(fileInput, './images/Untitled2.jpg');
    },
    'Click span "Upload"': function() {
        var span = $("#ContentHolder_btnUpload_CD").find(":containsExcludeChildren(Upload)");
        act.click(span);
    }
};
```

### PR DESCRIPTION
**Pull request by churkin**
\cc @inikulin @miherlosev

### CODE REVIEW

File: test/client/fixtures/utils/dom-test.js [hunk 1]
```diff
@@ -165,6 +165,26 @@ test('closest element', function () {
 
 module('isIframeWithoutSrc');
 
+asyncTest('After the location is set to an iframe without src', function () {
```
Comment (inikulin) [inline test/client/fixtures/utils/dom-test.js]: We use lowercased names for the client tests. The test name is unclear

**Comment by inikulin** [comment]
\r-

**Comment by churkin** [comment]
FPR

**Comment by inikulin** [comment]
lgtm

**Comment by miherlosev** [comment]
lgtm

**Pull request merged**